Test/LoaderTests.cs

```diff
@@ -1,14 +1,29 @@
+using System.IO;
+using FluentAssertions;
+using Hel.Tiled;
 using NUnit.Framework;
 
 namespace Test
 {
     [TestFixture]
     public class LoaderTests
     {
+        
         [Test]
-        public void PlaceholderTest()
-        {
-            Assert.Pass();
+        [TestCase("./MapFiles/beach_tileset.json")]
+        public void Should_LoadTileset_When_ValidPathProvided(string path) {
+            var tileset = Loader.LoadTileset(path);
+
+            tileset.Columns.Should().Be(36);
+            tileset.Image.Should().Be("beach_tileset.png");
+            tileset.Name.Should().Be("beach_tileset");
         }
+
+        [Test]
+        [TestCase("invalid_path.json")]
```

Comment:
is this how you tell it doesnt' work? doesn't it do it automatically

Reply:
`TestCase` is a way of passing parameters/values directly to the method. 

So this method takes in a string, this `TestCase` passes in a string. Each `TestCase` is its own run of the test with new parameters. 

An important note is that `TestCase` values MUST be constants. This means you can only use compile time values (Values that are hardcoded into the application) and not runtime values (Values calculated while the application is running, such as method/function returns or non-const variables)